src/pages/questions/QuestionDisplay.jsx

```diff
@@ -62,6 +62,30 @@ const QuestionDisplay = ({ question, answerQuestion, resetFilters }) => {
   const valueTagQuestionsURL = getValueTagQuestionsURL(question);
   const valueTagExamplesURL = getValueTagExamplesURL(question);
 
+  React.useEffect(() => {
+
+    function handleShortCut(event) {
+      const preventShortCut = event.target.tagName.toUpperCase() === 'INPUT'
+      if (question && !preventShortCut) {
+        switch (event.keyCode) {
+          case 75:
+            answerQuestion({ value: -1, insightId: question.insight_id });
+            break;
+          case 79:
+            answerQuestion({ value: 1, insightId: question.insight_id });
+            break;
+          case 78:
+            answerQuestion({ value: 0, insightId: question.insight_id });
+            break;
+          default:
+            break;
+        }
+      }}
+
+    window.addEventListener( 'keydown', handleShortCut)
+    return () =>  window.removeEventListener('keydown', handleShortCut)
+  }, [question, answerQuestion])
```

Comment:
better to use question?.insight_id in the dependency array (as it is a string and  question is an object so will change more frequently). Also you may need to check if the value is not null at the begnning of the useEffect.